WebContent/jsp/common/allImports.jsp

```diff
@@ -0,0 +1,24 @@
+
+
+
+<!-- import: CSS -->
+<!-- plugin css-->
+<link href="./plugins/bootstrap.css" rel="stylesheet" />
+
+<!-- app css -->
+<link href="./assets/css/styles.css" rel="stylesheet" />
+
+
+<!-- import: JS -->
+<!-- plugin js-->
+<script type="text/javascript" src="./plugins/jquery.2.2.3.min.js"></script>
+<script type="text/javascript" src="./plugins/bootstrap.min.js"></script>
+
+
+<!-- app js -->
+<script>
+	var appContextPath = "${pageContext.request.contextPath}";
+</script>
+

```

Comment:
```
<script>
    var appContextPath = "${pageContext.request.contextPath}";
</script>
```


this is how you can get the project name dynamically -- after JSP turning into HTML

```
<script>
    var appContextPath = "/student_management-MVC-using-ajax";
</script>
```

try printing this **${pageContext.request.contextPath}** , u will understand